src/parser/rogue/outlaw/modules/Abilities.js

```diff
@@ -120,6 +135,7 @@ class Abilities extends CoreAbilities {
         },
         castEfficiency: {
           suggestion: true,
+          extraSuggestion: <>In most cases this should be considered part of your rotation, only below <SpellLink id={SPELLS.ROLL_THE_BONES.id} icon /> on priority. However you should delay using it to line it up with <SpellLink id={SPELLS.BLADE_FLURRY.id} icon /> when there is more than one target.</>,
```

Comment:
In my opinion, the first part is over the top. Its enough to write `Only delay XXX for AoE scenarios`. 
Not insisting on it tho

Reply:
That's a good idea, that's a much more succinct way of putting it